src/Microsoft.DotNet.Darc/src/DarcLib/MaestroApiBarClient.cs

```diff
@@ -401,5 +401,30 @@ public Task<Build> UpdateBuildAsync(int buildId, BuildUpdate buildUpdate)
         {
             return _barClient.Builds.UpdateAsync(buildUpdate, buildId);
         }
+
+        /// <summary>
+        ///     Creates a new goal or updates the existing goal (in minutes) for a Defintion in a Channel.
+        /// </summary>
+        /// <param name="channel">Name of channel. For eg: .Net Core 5 Dev</param>
+        /// <param name="definitionId">Azure DevOps DefinitionId.</param>
+        /// <param name="minutes">Goal in minutes for a Definition in a Channel.</param>
+        /// <returns>Async task.</returns>
+        public Task<Goal> SetGoalAsync(string channel, int definitionId, int minutes)
+        {
+            var jsonData = new GoalRequestJson(minutes: minutes);
+            return _barClient.Goal.CreateAsync(body : jsonData, channelName : channel, definitionId : definitionId);
```

Comment:
There is strange whitespace on this line that doesn't match line 427.  
```suggestion
            return _barClient.Goal.CreateAsync(body: jsonData, channelName: channel, definitionId: definitionId);
```